pyproject.toml

```
@@ -30,6 +30,7 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

We're also testing 3.13..!
```suggestion
    "Programming Language :: Python :: 3.12",
    "Programming Language :: Python :: 3.13",
```